packages/ui-components/src/components/DataTable/PreviewFilters/PreviewFilters.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+
+import { ParametersType } from '../editing/types';
+import { FilterTypeOptions } from './filters';
+
+export const PreviewFilters = ({ parameters, onChange }) => {
+  return (
+    <Grid container spacing={2}>
+      {parameters.map(p => {
+        const option = FilterTypeOptions.find(t => t.value === p.type);
+        if (option) {
+          const { FilterComponent } = option;
+          return (
+            <Grid key={p.id} item xs={Math.floor(12 / parameters.length)}>
```

Comment:
This means if there's one param it takes up the full width? Kind of odd. Would be better to do fixed width FilterComponents (maybe 1/4 of grid each) and just have the grid work normally where it fills up and then wraps.

Reply:
Good point 👍. Added.

Reply:
Good point 👍 Done it in this PR: https://github.com/beyondessential/tupaia/pull/4338